### PR TITLE
Add Top News Albania

### PR DESCRIPTION
**Pull request by julianxhokaxhiu**
It uses a proxy url to obtain the official Twitch streaming m3u8 playlist.

### CODE REVIEW

**Review by BellezaEmporium — APPROVED** [review]

File: streams/al.m3u [hunk 1]
```diff
@@ -29,6 +29,8 @@ http://198.244.188.94/panorama/livestream/playlist.m3u8
 https://deb10stream.duckdns.org/hls/stream.m3u8
 #EXTINF:-1 tvg-id="SitiShqipTV.al",Siti Shqip TV (720p) [Not 24/7]
 https://vs.sednastream.com:1936/sitishqip/sitishqip/playlist.m3u8
+#EXTINF:-1 tvg-id="Top-Channel.tv",Top News (720p)
```
Comment (BellezaEmporium) [inline streams/al.m3u:32]:
```suggestion
#EXTINF:-1 tvg-id="TopChannel.al",Top News (720p)
```

**Review by BellezaEmporium — DISMISSED** [review]
See above.

**Comment by Dum4G** [comment]
Can't confirm it to be a great implementation. Returns 5 links that either fail to load after "commercial break" or load sound only in VLC. SimpleTV failed to open it at all. So may not be a good choice on many devices and players

**Comment by julianxhokaxhiu** [comment]
> Can't confirm it to be a great implementation. Returns 5 links that either fail to load after "commercial break" or load sound only in VLC. SimpleTV failed to open it at all. So may not be a good choice on many devices and players

Can you please try it on another stream? I did try it on various ones and it works just fine, but the topnewsal channel atm is having issues even opening it on Twitch. Returning a list of streams should be fine as they are are ordered from best to worst so ideally if the first doesn't work, it fallbacks to the second and so on. Also the m3u8 returned is up to the standard so I think any player should be able to handle that.

**Review by Dum4G — APPROVED** [review]

**Review by BellezaEmporium — COMMENTED** [review]
2 things on my side : 

-> No adaptive playlist is returned by the Twitch website it seems. Is it a old variant of query-streamlink in the backend ?
-> My antivirus flags the website as "suspicious activities detected". Rather odd.

**Review by BellezaEmporium — APPROVED** [review]

**Comment by BellezaEmporium** [comment]
My implementation gives a Twitch bumper saying "Preparing your stream... Watch a stream in it's natural habitat" before going in the stream.

**Comment by BellezaEmporium** [comment]
https://q-s.bellezaemporium.serv00.net/iptv-query?url=https://www.twitch.tv/topnewsal

if you need a check.

**Comment by Dum4G** [comment]
Same implementation, same result

**Comment by julianxhokaxhiu** [comment]
Thank you a lot :)

**Comment by BellezaEmporium** [comment]
> Same implementation, same result

Seems like it's a Twitch-only issue.

**Pull request merged**